src/index.js

```diff
@@ -0,0 +1,73 @@
+import '../style/style.css';
+
+const homeInner = require('./home');
+const menuInner = require('./menu');
+const contactInner = require('./contact');
```

Comment:
Make use of `import` and `export` instead of `require` or `module.exports` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))